tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -56,6 +56,30 @@ TTreeProcessorMT::TTreeProcessorMT(TTree &tree) : treeView(tree) {}
 /// \param[in] entries List of entry numbers to process.
 TTreeProcessorMT::TTreeProcessorMT(TTree &tree, TEntryList &entries) : treeView(tree, entries) {}
 
+////////////////////////////////////////////////////////////////////////
+/// Divide input data in clusters, i.e. the workloads to distribute to tasks
+std::vector<ROOT::Internal::TreeViewCluster> TTreeProcessorMT::MakeClusters()
+{
+   std::vector<ROOT::Internal::TreeViewCluster> clusters;
+   const auto &fileNames = treeView->GetFileNames();
+   const auto nFileNames = fileNames.size();
+   const auto &treeName = treeView->GetTreeName();
+   for (auto i = 0u; i < nFileNames; ++i) {
```

Comment:
Hi Philippe, thank you for looking at the PR.
Just to be sure, you suggest adding a comment even if I need the index `i` at line 77?